epimodel/exports/epidemics_org.py

```diff
@@ -557,6 +536,14 @@ def process_export(
         cummulative_active_df,
     )
 
+    r_estimates_df: pd.DataFrame = pd.read_csv(
```

Comment:
FYI There are some helper functions in `epimodel/utils.py` like `read_csv` and `read_csv_smart` that try to cover the parsing/handle our corner cases.